backend/src/main/java/pl/jawa/psinder/dto/ChatDto.java

```diff
@@ -0,0 +1,19 @@
+package pl.jawa.psinder.dto;
+
+
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class ChatDto {
+
+    private long id;
```

Comment:
Why do you have breaks between fields while the rest of them do it in a trickle of after themselves?